docs/src/modules/components/AppSearch.js

```diff
@@ -161,6 +107,13 @@ export default function AppSearch() {
   const theme = useTheme();
   const userLanguage = useSelector(state => state.options.userLanguage);
 
+  React.useEffect(() => {
+    loadCSS(
```

Comment:
I think that we should keep the loadCSS as a singleton. Otherwise, it gets loaded a many time the component mounts.

Reply:
Can't reproduce that the css is loaded multiple times during docs interaction. Could you describe how you get the css to load multiple times?

Reply:
Sure:

1. Open https://deploy-preview-17451--material-ui.netlify.com/getting-started/templates/
1. Notice the css file load a first time
1. Open the dev tool to remove the `target="_blank"` from one of the card images.
1. Click on the image
1. Go back
1. Notice the css file loads a second time

Reply:
> Open the dev tool to remove the target="_blank" from one of the card images

So it's not reproducible for users?

Reply:
You probably disabled the cache in the devtools? I would expect it to load a second times when the cache is disabled as any other resource should.

Reply:
Should we ignore the following behavior then?

![Capture d’écran 2019-09-16 à 20 04 56](https://user-images.githubusercontent.com/3165635/64981872-459f5800-d8bd-11e9-87c3-6f52af2e1665.png)

Reply:
This seems like a but with `loadCSS`. I expected the second argument to be the target of the injection. Something else is adding the style tag.

Reply:
I got confused by `<style id />`. I assumed this was used to populate the styles but it's only a marker for loadCSS to insert a new! style node before.